### PR TITLE
[Docs] Add MultiImageMixDataset tutorial

### PR DESCRIPTION
**Pull request by RockeyCoss**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Add MultiImageMixDataset tutorial
Related PR：
https://github.com/open-mmlab/mmsegmentation/pull/1105 MultiImageMixDataset
https://github.com/open-mmlab/mmsegmentation/pull/1093 Mosaic transform
## Modification

Add MultiImageMixDataset tutorial

### CODE REVIEW

**Review by Junjun2016 — COMMENTED** [review]
Add related PRs to PR messages, e.g. MultiImageMixDataset and Mosaic.

**Review by Junjun2016 — APPROVED** [review]

File: docs/en/tutorials/customize_datasets.md [hunk 1]
```diff
@@ -172,3 +172,39 @@ data = dict(
...
+### Multi-image Mix Dataset
+
+We use `MultiImageMixDataset` as a wrapper to mix images from multiple datasets.
+`MultiImageMixDataset` can be used by multiple images mixed data augmentation
+like mosaic and mixup.
+
+An example of using `MultiImageMixDataset` with `Mosaic` data augmentation:
+```python
+train_pipeline = [
+    dict(type='Mosaic'),
```
Comment (lkm2835) [inline docs/en/tutorials/customize_datasets.md:185]: `dict(type='Mosaic')`
-> `dict(type='RandomMosaic', prob=1)`

I think it's better to modify it.

File: docs/zh_cn/tutorials/customize_datasets.md [hunk 2]
```diff
@@ -171,3 +171,38 @@ data = dict(
...
+
+### 多图混合集
+
+我们使用 `MultiImageMixDataset` 作为包装(wrapper)去混合多个数据集的图片。
+`MultiImageMixDataset`可以被类似mosaic和mixup的多图混合数据増广使用。
+
+`MultiImageMixDataset`与`Mosaic`数据増广一起使用的例子：
+```python
+train_pipeline = [
+    dict(type='Mosaic'),
```
Comment (lkm2835) [inline docs/zh_cn/tutorials/customize_datasets.md:183]: `dict(type='Mosaic')`
-> `dict(type='RandomMosaic', prob=1)`

File: docs/en/tutorials/customize_datasets.md (hunk 1 above)
Comment (lkm2835) [inline docs/en/tutorials/customize_datasets.md:185]: `dict(type='Mosaic')`
-> `dict(type='RandomMosaic', prob=1)`

I think it's better to modify it.

File: docs/zh_cn/tutorials/customize_datasets.md (hunk 2 above)
Comment (lkm2835) [inline docs/zh_cn/tutorials/customize_datasets.md:183]: `dict(type='Mosaic')`
-> `dict(type='RandomMosaic', prob=1)`

**Review by lkm2835 — CHANGES_REQUESTED** [review]

**Comment by MengzhangLI** [comment]
It is ok except `dict(type='Mosaic')` may be modified to `dict(type='RandomMosaic', prob=1)`.

**Review by MengzhangLI — APPROVED** [review]

**Review by MengzhangLI — APPROVED** [review]

File: docs/en/tutorials/customize_datasets.md [hunk 3]
```diff
@@ -172,3 +172,39 @@ data = dict(
...
 ```
+
+### Multi-image Mix Dataset
+
+We use `MultiImageMixDataset` as a wrapper to mix images from multiple datasets.
+`MultiImageMixDataset` can be used by multiple images mixed data augmentation
+like mosaic and mixup.
+
+An example of using `MultiImageMixDataset` with `Mosaic` data augmentation:
+```python
```
Comment (MeowZheng) [inline docs/en/tutorials/customize_datasets.md:183]: image

MD031/blanks-around-fences: Fenced code blocks should be surrounded by blank lines

File: docs/zh_cn/tutorials/customize_datasets.md [hunk 4]
```diff
@@ -171,3 +171,38 @@ data = dict(
...
 
 ```
+
+### 多图混合集
+
+我们使用 `MultiImageMixDataset` 作为包装(wrapper)去混合多个数据集的图片。
+`MultiImageMixDataset`可以被类似mosaic和mixup的多图混合数据増广使用。
+
+`MultiImageMixDataset`与`Mosaic`数据増广一起使用的例子：
+```python
```
Comment (MeowZheng) [inline docs/zh_cn/tutorials/customize_datasets.md:181]: MD031/blanks-around-fences: Fenced code blocks should be surrounded by blank lines

**Review by MeowZheng — APPROVED** [review]

**Pull request merged**